### PR TITLE
Create 5.8.1 release notes

### PR DESCRIPTION
**Pull request by nkolev92**
Related https://github.com/NuGet/Client.Engineering/issues/695

Note that this needs to be merged by 1/12 cc @JonDouglas @aortiz-msft

### CODE REVIEW

File: docs/release-notes/NuGet-5.8.md [hunk 1]
```diff
@@ -136,6 +137,15 @@ Thank you to all the contributors who helped make this NuGet release awesome!
 [Therzok](https://github.com/Therzok) | [3636](https://github.com/NuGet/NuGet.Client/pull/3636) | `RuntimeEnvironmentHelper.IsRunningOnVisualStudio` performance issue on Mono - [#9989](https://github.com/NuGet/Home/issues/9989)
 [thomaslevesque](https://github.com/thomaslevesque) | [3442](https://github.com/NuGet/NuGet.Client/pull/3442) | Add a TypeConverter for the SemanticVersion class - [#9125](https://github.com/NuGet/Home/issues/9125)
 
+## Summary: What's New in 5.8.1
+
+* packages.config package.lock.json uses an incorrect target framework in 5.8 - [#10257](https://github.com/NuGet/Home/issues/10257)
+
+* 5.8 + 16.8 Cannot resolve transitive project dependencies when mixing PackageReference and packages.config - [#10326](https://github.com/NuGet/Home/issues/10326)
+
+**[List of all issues fixed in this release - 5.8.1](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5ff7aeae16150e3b19910391)**
```
Comment (JonDouglas) [inline docs/release-notes/NuGet-5.8.md:146]: Will this link work for the public?

Comment (nkolev92) [reply docs/release-notes/NuGet-5.8.md:146]: Only if you have zenhub installed. 

I followed the pattern from the 5.8 release notes.

Comment (JonDouglas) [reply docs/release-notes/NuGet-5.8.md:146]: It won't, but we also have it for 5.8. 

Maybe in 5.9 we export a CSV or use a different tool to generate all issues fixed similar to VS Code?

https://vscode-tools.azurewebsites.net/

Just gonna approve ready for merge.

**Review by JonDouglas — APPROVED** [review]
LGTM, let's fix zenhub links in 5.9 moving forward

**Pull request merged**